unit_test/test_spdm_requester/challenge.c

```diff
@@ -1920,6 +1920,13 @@ void libspdm_test_requester_challenge_case2(void **state)
         SPDM_CHALLENGE_REQUEST_NO_MEASUREMENT_SUMMARY_HASH,
         measurement_hash, NULL);
     assert_int_equal(status, LIBSPDM_STATUS_SUCCESS);
+
+    /* Completion of CHALLENGE sets M1/M2 to null. */
+#if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
+    assert_int_equal(spdm_context->transcript.message_c.buffer_size, 0);
+#else
+    assert_int_equal(spdm_context->transcript.digest_context_m1m2, NULL);
```

Comment:
Use [assert_null](https://api.cmocka.org/group__cmocka__asserts.html#ga42bcfbd60054ba3e64d94fa28ba5565c).

Reply:
Thanks, I have updated to use [assert_null](https://api.cmocka.org/group__cmocka__asserts.html#ga42bcfbd60054ba3e64d94fa28ba5565c).